src/pkg/services/m365/m365_test.go

```diff
@@ -102,6 +102,38 @@ func (suite *M365IntegrationSuite) TestGetUserInfo() {
 	assert.Equal(t, "user", info.Mailbox.Purpose)
 }
 
+func (suite *M365IntegrationSuite) TestUserHasMailbox() {
+	ctx, flush := tester.NewContext()
+	defer flush()
+
+	var (
+		t    = suite.T()
+		acct = tester.NewM365Account(t)
+		uid  = tester.M365UserID(t)
+	)
+
+	enabled, err := m365.UserHasMailbox(ctx, acct, uid)
+	require.NoError(t, err, clues.ToCore(err))
+	require.NotNil(t, enabled)
+	require.True(t, *enabled)
```

Comment:
Prefer assert unless the following checks will be unable to run if the current check fails.
```suggestion
	assert.NotNil(t, enabled)
	assert..True(t, *enabled)
```